lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -0,0 +1,538 @@
+  process.env.ExecutionsTable = cryptoRandomString({ length: 10 });
+  process.env.GranulesTable = cryptoRandomString({ length: 10 });
+
+  collectionsModel = new Collection();
+  await collectionsModel.createTable();
+
+  executionsModel = new Execution();
+  await executionsModel.createTable();
+
+  granulesModel = new Granule();
+  await granulesModel.createTable();
+
+  t.context.granulePgModel = new GranulePgModel();
+  t.context.filePgModel = new FilePgModel();
+
+  // Store the CMR password
+  process.env.cmr_password_secret_name = cryptoRandomString({ length: 5 });
+  await secretsManager().createSecret({
+    Name: process.env.cmr_password_secret_name,
+    SecretString: cryptoRandomString({ length: 5 }),
+  }).promise();
+
+  const { knexAdmin, knex } = await generateLocalTestDb(
+    testDbName,
+    migrationDir
+  );
+  t.context.knexAdmin = knexAdmin;
+  t.context.knex = knex;
+});
+
+test.beforeEach(async (t) => {
+  const collectionPgModel = new CollectionPgModel();
+  const testCollection = fakeCollectionRecordFactory();
+
+  const collectionResponse = await collectionPgModel.create(
+    t.context.knex,
+    testCollection
+  );
+  t.context.testCollection = testCollection;
+  t.context.collectionCumulusId = collectionResponse[0];
+  t.context.collectionPgModel = collectionPgModel;
+
+  const executionPgModel = new ExecutionPgModel();
+  const testExecution = fakeExecutionRecordFactory();
+
+  const executionResponse = await executionPgModel.create(
+    t.context.knex,
+    testExecution
+  );
+  t.context.testExecution = testExecution;
+  t.context.executionCumulusId = executionResponse[0];
+  t.context.executionPgModel = executionPgModel;
+
+  t.context.testGranule = generateTestGranule(testCollection, testExecution.arn);
+});
+
+test.afterEach.always(async (t) => {
+  await t.context.knex(tableNames.files).del();
+  await t.context.knex(tableNames.granules).del();
+  await t.context.knex(tableNames.collections).del();
+  await t.context.knex(tableNames.executions).del();
+});
+
+test.after.always(async (t) => {
+  await granulesModel.deleteTable();
+  await collectionsModel.deleteTable();
+  await executionsModel.deleteTable();
+
+  await s3Utils.recursivelyDeleteS3Bucket(bucket);
+
+  await secretsManager().deleteSecret({
+    SecretId: process.env.cmr_password_secret_name,
+    ForceDeleteWithoutRecovery: true,
+  }).promise();
+
+  await destroyLocalTestDb({
+    ...t.context,
+    testDbName,
+  });
+});
+
+test.serial('migrateGranuleRecord correctly migrates granule record', async (t) => {
+  const {
+    collectionCumulusId,
+    executionCumulusId,
+    granulePgModel,
+    knex,
+    testGranule,
+  } = t.context;
+
+  await migrateGranuleRecord(testGranule, knex);
+
+  const record = await granulePgModel.get(knex, {
+    granule_id: testGranule.granuleId,
+    collection_cumulus_id: collectionCumulusId,
+  });
+
+  t.deepEqual(
+    omit(record, ['updated_at', 'cumulus_id']),
```

Comment:
The value of `updated_at` should be predictable, so I don't know why we need to omit it. And if you use `t.like` instead of `t.deepEqual`, you don't need to bother with the `omit` because it only compares the properties that exist in the second object

Reply:
Yeah, that is true. I am opting to use `deepEqual` though for this case because I want to ensure I am accounting for all fields in the object. I had initially used `t.like` but changed my mind here https://github.com/nasa/cumulus/pull/2051#discussion_r563918827 

For clarity, you are correct that I don't need to omit `updated_at`

Reply:
[`6ad9950` (#2051)](https://github.com/nasa/cumulus/pull/2051/commits/6ad99508c7ee725ec8bd8b4c40ca3092192275f4)